float64 speed      # Speed between 0.0 - 1.0 (low - high)
---